source/hdev/Camera/Basic/CaptureHDRCompleteSettings.hdev

```diff
@@ -50,7 +50,6 @@
 <l>set_framegrabber_param (AcqHandle, 'Brightness', 1)</l>
 <l>set_framegrabber_param (AcqHandle, 'Gain', 1)</l>
 <c></c>
-<l>set_framegrabber_param (AcqHandle, 'PatternsSineBidirectional', 0)</l>
```

Comment:
There are similar lines further down in this file.

Reply:
Just search the whole repo for "bidirectional" :)

Reply:
Done, I was testing with the wrong version. Should be ok now.

Reply:
Looks good. No more bidirectional references :)